<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>carregaEstilosFrame</class>
 <widget class="QFrame" name="carregaEstilosFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>321</width>
    <height>184</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Carregar Estilos</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="gifLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="mainFrame">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Base de Dados :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="bancoCombo"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Carta :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cartaCombo"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Tipo de Estilo :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="estiloCombo"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="carregarBotao">
          <property name="text">
           <string>Carregar</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
